packages/noya-state/src/selectors/pointSelectors.ts

```diff
@@ -7,6 +8,20 @@ import { visit } from '../layers';
 import { getCurrentPage } from './pageSelectors';
 import { getBoundingRectMap } from './selectors';
 
+export const POINT_RADIUS = 4;
+
+export const isPointInRange = (point: Point, rawPoint: Point): boolean => {
```

Comment:
It's probably not noticeable at this size, but wouldn't this select a square instead of a circle? Would this be more accurate?

```ts
distance(point, rawPoint) < POINT_RADIUS
```